standards.md

```diff
@@ -36,22 +36,17 @@ _See also [the Tekton review process](https://github.com/tektoncd/community/blob
 
 ## Commits
 
-* Follow [commit messages best practices](https://chris.beams.io/posts/git-commit/):
-  1. Separate subject from body with a blank line
-  2. Limit the subject line to 50 characters
-  3. Capitalize the subject line
-  4. Do not end the subject line with a period
-  5. Use the imperative mood in the subject line
-  6. Wrap the body at 72 characters
-  7. Use the body to explain what and why vs. how. Don't just link to an issue and
-      [aim for 2 paragraphs](https://www.youtube.com/watch?v=PJjmw9TRB7s), e.g.:
-      * What is the problem being solved?
-      * Why is this the best approach?
-      * What other approaches did you consider?
-      * What side effects will this approach have?
-      * What future work remains to be done?
+* Use the body to explain [what and why vs. how](https://chris.beams.io/posts/git-commit/#why-not-how).
+  Link to an issue whenever possible and [aim for 2 paragraphs](https://www.youtube.com/watch?v=PJjmw9TRB7s),
+  e.g.:
+  * What is the problem being solved?
+  * Why is this the best approach?
+  * What other approaches did you consider?
+  * What side effects will this approach have?
+  * What future work remains to be done?
 * Prefer one commit per PR; if there are multiple commits, ensure each is
-  self-contained and makes sense without the context of the others in the PR
+  self-contained and makes sense without the context of the others in the PR.
+* For useful guidance writing commits take a look at [these seven commit message best practices](https://chris.beams.io/posts/git-commit/#seven-rules).
```

Comment:
While having the link is fine, I think most people will stop at what is here, as they had to click a link to get here already, and they won't click a 2nd time most likely.
Some of the recommendations might feel like nit-picks but I'm not happy about removing (1) (2) and (6) as it makes the git history harder to use if you're not using a graphical interface.

Reply:
I've brought those 3 items back.  I also added a template that hopefully exemplifies what we want so that users in a rush or struggling to parse the rules can just copy/paste it and edit.